src/rpc/request.cpp

```diff
@@ -98,12 +111,22 @@ bool GenerateAuthCookie(std::string *cookie_out)
     file << cookie;
```

Comment:
Should probably set permissions before writing to the file (but after opening it, so we already have write access), just in case they are more restrictive?